<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(202, 202, 202);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Gen Hash</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>771</width>
     <height>61</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>30</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>sha256</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha512</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha384</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha224</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha3_384</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha3_224</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>md5</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Brute</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>30</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>sha256</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha512</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha384</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha224</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha3_384</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sha3_224</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>md5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Хеширование пароля</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Брутфорс Пароля</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
